<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filterCanny</class>
 <widget class="QWidget" name="filterCanny">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>175</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lbSigma">
     <property name="text">
      <string>Sigma</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbLowThreshold">
     <property name="text">
      <string>Low Threshold</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leSigma">
     <property name="text">
      <string notr="true">1.0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbHighThreshold">
     <property name="text">
      <string>High Threshold</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="leLowThreshold">
     <property name="inputMask">
      <string>009</string>
     </property>
     <property name="text">
      <string>30</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="leHighThreshold">
     <property name="inputMask">
      <string>009</string>
     </property>
     <property name="text">
      <string>80</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
